Docs

serpi90 opened the pull request:
Recreated from #15 because master was renamed to release-candidate
- [x] Adds an example command line handler.
- [x] Adds Travis tests using the example command line handler.
- [x] closes: #11 referring to example command line handler.
- [x] closes: #14

serpi90 committed:
feature: add FlagArgument for boolean arguments whose value depends on being present or not.

serpi90 committed:
feature: add example handler for use in documentation and tests outside the image.

serpi90 committed:
fix: add --quit to example for use in tests

serpi90 committed:
feature: add travis tests for the example

serpi90 committed:
fix: baseline loading, and working directory not working on Pharo 6

serpi90 committed:
refactor: remove option to set FlagArguments as true by default as it is misleading

gcotelli commented:
@serpi90 :  Did you try something like https://github.com/kward/shunit2 ? I know it's used in smalltalkCI to test the bash scripts.

serpi90 commented:
I know there are shell testing frameworks, but I only plan to write that short suite, and a simple file should work.

The functions cloud be moved to another file if tests were more extensive, but it's not the case right now.

serpi90 committed:
refactor: remove --quit from example

serpi90 committed:
refactor: extract test script utils to another file for readability

add test for Error cases

serpi90 committed:
doc: Usage examples and explain --debug-mode and --suspend-ui

serpi90 committed:
Revert "refactor: remove --quit from example"

This reverts commit 57b11c05d106c448586f3e81b9b8793dd725c8ad.

serpi90 committed:
doc: spelling and grammar fixes

serpi90 committed:
doc: improve docs

- rename addend to add
- moved arguments docs to class comments
- place implicit arguments above usage example

serpi90 committed:
doc: add headers to HowTo

this should improve the section "Launching your application using the handler".

gcotelli commented on docs/HowTo.md:
```diff
@@ -1,36 +1,144 @@
+[2019-05-31T03:06:56.962318-03:00] [INFO] add: 40
+[2019-05-31T03:06:56.963318-03:00] [INFO] fail: false
+[2019-05-31T03:06:56.964318-03:00] [INFO] raise-error: false
+[2019-05-31T03:06:56.965318-03:00] [WRANING]  seed option not provided. Defaulting to 0"
```
Remove the last "
